R/ansi.R

```diff
@@ -1,5 +1,5 @@
-# nocov start - https://github.com/tidyverse/tibble/blob/main/R/fansi.R
-set_fansi_hooks <- function() {
+# nocov start - https://github.com/tidyverse/tibble/blob/main/R/ansi.R
+set_ansi_hooks <- function() {
```

Comment:
```suggestion
set_fansi_hooks <- function() {
```

Probably better not to rename the function since it is used (even if not exported)